src/Core/Utils/SystemUtils.hpp

```
@@ -93,4 +101,55 @@ namespace SystemUtils {
+
+        auto procAddr = (void *)GetProcAddress(handle, "vkGetInstanceProcAddr");
+        FreeLibrary(handle);
+
+        if (!procAddr)
+            return { false,VkLoaderDiag::CANNOT_BE_LOADED };
+
+        return { true };
+
+
+#elif defined(__linux__) || defined(__APPLE)
+        void *handle = dlopen(VULKAN_LOADER, RTLD_NOW | RTLD_LOCAL);
```

Clang (14.0.6)

[warning] variable 'handle' is not initialized

(cppcoreguidelines-init-variables)